packages/docs/content/12-enterprise-version/00-on-prem-system-requirements.md

```diff
@@ -0,0 +1,284 @@
+| | * Read service configuration
+| | Snapshot management
+| | * Create snapshot
+| | * Remove snapshot
+| | * Rename snapshot
+| | * Revert to snapshot
+| | vSphere Replication
+| | * Configure replication
+| | * Manage replication
+| | * Monitor replication
+| vApp | Import
+| | View OVF environment
+| | vApp application configuration
+| | vApp instance configuration
+| vSphere Tagging| Create vSphere Tag
+| | Edit vSphere Tag
+
+
+### Optional Settings 
+The requirement configurations given below are optional.
+
+* DNS Mapping
+   - Static IP DNS: [Name servers] Comma separated DNS addresses (e.g: 8.8.8.8, 192.168.0.8), required only for static IP allocation.
+* SMTP Settings
+   - Configure SMTP settings to enable the Spectro Cloud platform to send out email notifications. Email Notifications are sent out to new users when they are on boarded to the platform so they can activate their accounts.
+* Certificate (optional)
+   - Provide the desired SSL or TLS server certificates to support external access to valid HTTPs.
+* FTP Location (backups)
+   - An accessible FTP location to backup data.
+   	
+# Quick Start Mode
+
+The Spectro Cloud On-Prem Quick Start Mode is a single node installation of the Spectro Cloud platform. This deployment is used to quickly understand the capabilities of the Spectro Cloud platform. For production purposes the enterprise version is recommended. The minimum requirements for Quick Start version is outlined below:
+
+### Hardware Requirements 
+
+Hardware requirements will describe the CPU, memory, and disk requirements for the nodes.
+* 8GB memory
+* 4 virtual CPUs 
+* Secondary storage of 80 GB distributed across multiple HDDs.
+
+### Network Requirements
+
+* Quick Start will operate on a single instance which requires connectivity through a proxy or non-proxy environment. 
+* This instance can be configured with static IP addresses or dynamically allocated IP addresses.
+
+<InfoBox>
+ Make sure that your Datacenter CIDR IP address does not overlap Kubernetes PodCIDR range. Kubernetes PodCIDR range settings can be changed while installing Quick Start Version
+</InfoBox>
+
+# Enterprise Mode
+
+The Spectro Cloud On-Premise Enterprise version is a highly available multinode infrastructure for production purposes. The minimum number of nodes/VMs required by this environment is 3. Depending on the number of nodes the requirements changes. The minimum requirements are outlined below:
+
+### Hardware Requirements 
+Hardware requirements will describe the CPU, memory, and disk requirements for the nodes.
+* Each VM requires 8GB memory resulting in 24GB RAM in total .
+* 4 vCPUs per node * 3 nodes requiring 12 vCPUs
+* Secondary storage of 240GB distributed across multiple HDDs.
+
+### Network Requirements
+
+* A Range or a Subnet of at least 5 static IP addresses is required for the installation and ongoing management. 
+* The infrastructure will require a proxy/non proxy environment for accessibility. 
```

Comment:
The instances should be reachable to the whitelisted domains either through Proxy or Without Proxy Network Connections.